nautobot/core/ui/object_detail.py

```diff
@@ -1053,6 +1104,33 @@ def get_extra_context(self, context):
         }
 
 
+class _ObjectCommentPanel(ObjectFieldsPanel):
```

Comment:
This should be changed to a TextPanel (or MarkdownPanel, or equivalent) subclass as part of @mateusz-lachowski-codilime work on #6139.

Reply:
Maybe if this PR gets merge in before https://github.com/nautobot/nautobot/issues/6139. is complete then we can add it as part of its acceptance criteria